scripts/post-changelog.mjs

```diff
@@ -1,17 +1,15 @@
 import { $, fetch, argv } from "zx";
 import dotenv from "dotenv-safe";
-import conventionalChangelog from "conventional-changelog";
-import { getPackages } from "@lerna/project";
-import slackify from "slackify-markdown";
+import slackifyMarkdown from "slackify-markdown";
 import { simpleGit } from "simple-git";
 import gitDiffParser from "gitdiff-parser";
 
 /* eslint-disable no-console */
 
-const CHANNEL = `orbit-react`;
-const COLOR_CORE = `#00A58E`;
-const PACKAGES = ["orbit-components", "orbit-tailwind-preset"];
+const CHANNEL = "orbit-react";
 const PACKAGE_PREFIX = "@kiwicom";
+const COLOR_CORE = "#00A58E";
+const PACKAGES = ["orbit-components", "orbit-tailwind-preset"];
```

Comment:
This can be increased as we like. I believe these two are the most relevant to end users, but correct me if I'm wrong.

Reply:
orbit-design-tokens should be included also then :)